src/sentry/integrations/slack/views/link_team.py

```diff
@@ -69,23 +78,36 @@ class SlackLinkTeamView(BaseView):
     Django view for linking team to slack channel. Creates an entry on ExternalActor table.
     """
 
+    _METRICS_SUCCESS_KEY = SLACK_BOT_COMMAND_LINK_TEAM_SUCCESS_DATADOG_METRIC
+    _METRICS_FAILURE_KEY = SLACK_BOT_COMMAND_LINK_TEAM_FAILURE_DATADOG_METRIC
+
     @method_decorator(never_cache)
     def handle(self, request: Request, signed_params: str) -> HttpResponseBase:
         if request.method not in ALLOWED_METHODS:
             return render_error_page(request, status=405, body_text="HTTP 405: Method not allowed")
 
         try:
-            params = unsign(signed_params)
-        except (SignatureExpired, BadSignature):
+            converted = unsign(signed_params)
+            params = TeamLinkRequest(**converted)
+        except (SignatureExpired, BadSignature) as e:
+            _logger.warning("handle.signature_error", exc_info=e)
```

Comment:
should we specify that this is slack handling in the logger

Reply:
we should be able to get based on how we initialized the logger right `__name__`?

Reply:
that's true